### PR TITLE
Zapier instructions

### PR DESCRIPTION
**Pull request by afoonalex**

### CODE REVIEW

File: user-guide/integrations/zapier-integration.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Creating a Zap in Zapier for New Query Results
+
+You can connect Redash to Zapier and trigger an action every time a new line is added to you query results - for example, a Slack bot that announces every time a new order came in.
+The best way is to have a scheduled query that refreshes frequently so the notification you get is in real-time.
+
+For now you'll need to ask us for an invitation as it's still in beta - you can do that via Intercom (at the bottom right corner of every page in /help or inside the Redash webapp or [Slack](https://redash.slack.com/).
```
Comment (arikfr) [inline user-guide/integrations/zapier-integration.md]: 1. The link to get an invite to Slack is slack.redash.io (they can't sign in from the other one).
2. Let's just post the Zapier link here... I checked again and didn't see anything explicitly prohibiting this :)

File: user-guide/integrations/zapier-integration.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+Here's a short example of connecting Redash to Zapier and creating a new Zap:
+1. Open the invitation you got via email to use the beta Redash trigger app (,create an account if needed) and click "Make a Zap" right next to your account area at the top of the page.
+
+2. Choose a trigger app - in this case, Redash (it won't be available to you unless you have received and opened the invitation)
+
+3. Choose a trigger - for now we have only "New Query Results" in there, this means the Zap will get triggered everytime a new line is added to the results set of your query.
```
Comment (arikfr) [inline user-guide/integrations/zapier-integration.md]: Worth reiterating that Zapier checks for existing results, so they should schedule the query to refresh according to the frequency they defined Zapier to check for results.

File: user-guide/integrations/zapier-integration.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+4. Connect your Redash account:
+
+  4.1. Full domain of your Redash account (Something like: `https://app.redash.io/myorg/` or `https://redash.myorg.com/` (note the slash at the end))
+
+  4.2. Your User ID - go to your account in Redash, the number in the URL just after /users/ is your User ID, i.e. `https://app.redash.io/myorg/users/{user id}`
+
+  4.3. API key - also in your user settings, select the API KEY tab and copy it from there
+
+  4.4. Test your connection!
```
Comment (arikfr) [inline user-guide/integrations/zapier-integration.md]: How about a screenshot of this?

File: user-guide/integrations/zapier-integration.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+5. Select the query you want to get the Zap for by entering the query ID (`https://app.redash.io/myorg/queries/{query id}`)
+
+6. Select the app you want to perform an action when a new query result is added to your query - Slack is a good option.
+
+7. Select an action - for this example we'll use "Send Channel Message". You'll need to connect your Slack account to Zapier as well, if you are logged-in in your browser it'll be pretty instant.
+
+8. Enter the channel, message text (you can select different column values from your query), bot/your user in slack, bot name, bot icon emoji or icon url, auto expend links and mentions settings. Bots are cool so you should definitely send the zap as a bot!
```
Comment (arikfr) [inline user-guide/integrations/zapier-integration.md]: And a screenshot of this defined so it's clearer what they see in the Slack screenshot.

File: user-guide/integrations/zapier-integration.md (hunk 1 above)
Comment (afoonalex) [inline user-guide/integrations/zapier-integration.md]: k - can I use the link I got via email? is it universal?

File: user-guide/integrations/zapier-integration.md (hunk 3 above)
Comment (afoonalex) [inline user-guide/integrations/zapier-integration.md]: which part?  the connection test in zapier? it's not an interesting screen, there are so many steps and each one is tiny, if you have the instructions open in one tab and zapier in another you'll be good even if it's textual imo

File: user-guide/integrations/zapier-integration.md (hunk 4 above)
Comment (afoonalex) [inline user-guide/integrations/zapier-integration.md]: zapier_slack_template
this? it's looooong

Comment (afoonalex) [inline user-guide/integrations/zapier-integration.md]: prepared a wider version 
zapier_slack_template - wider

**Comment by afoonalex** [comment]
waiting for your input regarding my questions up there ^ @arikfr

File: user-guide/integrations/zapier-integration.md (hunk 1 above)
Comment (arikfr) [inline user-guide/integrations/zapier-integration.md]: Not sure what you got via email. I meant this one: https://zapier.com/developer/invite/32785/4910e4da7931a8f3a2124ebd85cc352b/

File: user-guide/integrations/zapier-integration.md (hunk 3 above)
Comment (arikfr) [inline user-guide/integrations/zapier-integration.md]: The screen where you select Redash and enter API key/user id.

File: user-guide/integrations/zapier-integration.md (hunk 4 above)
Comment (arikfr) [inline user-guide/integrations/zapier-integration.md]: We can use the left part of the wide version, as it includes all the interesting parts.

**Comment by arikfr** [comment]
Inputed.

**Pull request merged**